### DISCUSSION TITLE
Firmware For Red Sampler

### DISCUSSION THREAD

**Discussion by GFce** [bangcorrupt/hacktribe#158] [Q&A] [answered]
Hi

Brand new to this and looks great. Is there anywhere for someone not very familiar with this stuff (like me) to download a finalised file or get one via email to update the Electribe with? I have the later red sampler version.

Thanks
G

**Comment by bangcorrupt** [comment, marked as answer]
Use the [GUI](https://github.com/bangcorrupt/hacktribe/wiki/How-To#patch-with-a-gui).  Post in [this thread](https://github.com/bangcorrupt/hacktribe/discussions/41) if you get stuck.

**Reply by Piergap** [reply]
i'm working on a tutorial. Just made the hack a couple of weeks ago, and i see that it would help a lot of people
i'll post a link as soon as done!

**Comment by Piergap** [comment]
as promised, uploaded on youtube +  links to the final patched system file and tools (in the video description) 

https://www.youtube.com/watch?v=Ep0c7jM9d3M
